osu.Game/Overlays/OverlayView.cs

```diff
@@ -18,6 +18,11 @@ namespace osu.Game.Overlays
     public abstract class OverlayView<T> : CompositeDrawable, IOnlineComponent
         where T : class
     {
+        /// <summary>
+        /// Whether we should perform fetch on api state change to online (true by default).
+        /// </summary>
+        protected virtual bool PerformFetchOnApiStateChange => true;
```

Comment:
not sure I see the point. this flag essentially guts the only actual non-obvious shared logic `OverlayView` adds. the rest might as well be duplicated locally since it's obvious.

the problem here is trying to reuse `OverlayView` which is tied to login state, for something that *shouldn't* be tied to login state.

Reply:
I guess, making `FrontPageDisplay` inherit `OverlayView` was a bad decision?

Reply:
that would be the logical conclusion to me, yes.

Reply:
Will change it back then